### DISCUSSION TITLE
[Bug]: Frigate+ UI metrics display wrong data

### DISCUSSION THREAD

**Discussion by FabianGabor** [blakeblackshear/frigate#13491] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

![Screenshot 2024-09-01 at 21-09-41 Frigate Premium Services for Frigate NVR](https://github.com/user-attachments/assets/e17e123c-706d-43de-89a5-525ee9524fee)

I don't have any labels for amazon, deer, fedex, package, ups. I might have a few license_plate labels but definetly not that many (maybe 10-50).
Pretty sure that the cat, car, dog and person count is not the same.

Would like to get back the verified/total images counter.

### Steps to reproduce

Open https://plus.frigate.video/dashboard/

### Version

2024-09-01-20-14 UTC

### In which browser(s) are you experiencing the issue with?

Firefox 129.0.2 (64-bit)

### Frigate config file

```yaml
N/A
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

Other

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

N/A

### Screenshots of the Frigate UI's System metrics pages

N/A

**Comment by NickM-27** [comment]
This is not a bug, this data was not kept previously so it shows the number of images for all of them, this is working as expected. 

Also, this is not the place to submit issues for frigate+ website as this is for frigate itself.

**Reply by FabianGabor** [reply]
> this data was not kept previously so it shows the number of images for all of them

Will those numbers be recounted at some point? Because otherwise it's just bad data.

I figured this isn't the right place but https://github.com/blakeblackshear?tab=repositories&q=frigate doesn't list any public repo to for frigate+. Where's would be the right place for submitting issues/feedback/feature requests?

**Reply by NickM-27** [reply, marked as answer]
I think you are misunderstanding what the counts mean. It is not saying how many of each label have been annotated. It is saying the number of images that have been checked and verified for that label.

In general emailing blake@frigate.video is likely the best way to go about it
